pulsar-functions/instance/src/main/python/python_instance.py

```diff
@@ -172,6 +172,7 @@ def run(self):
           consumer_type=mode,
           message_listener=partial(self.message_listener, self.input_serdes[topic]),
           unacked_messages_timeout_ms=int(self.timeout_ms) if self.timeout_ms else None,
+          receiver_queue_size=int(consumer_conf.receiverQueueSize.receiverQueueSize) if consumer_conf.receiverQueueSize else 1000,
```

Comment:
shouldn't we just not set it and fall back to the system default what ever value that may be?